sql/core/src/main/scala/org/apache/spark/sql/jdbc/PostgresDialect.scala

```diff
@@ -66,6 +70,7 @@ private object PostgresDialect extends JdbcDialect {
     case BooleanType => Some(JdbcType("BOOLEAN", Types.BOOLEAN))
     case FloatType => Some(JdbcType("FLOAT4", Types.FLOAT))
     case DoubleType => Some(JdbcType("FLOAT8", Types.DOUBLE))
+    case ShortType => Some(JdbcType("SMALLINT", java.sql.Types.SMALLINT))
     case t: DecimalType => Some(
```

Comment:
Hi @JoshRosen, do you mind if I ask a question please (although it is a super minor)? It seems `java.sql.Types.SMALLINT` and `Types.SMALLINT` are pointing the same constant. Are there a reason to do this like this?